packages/view/backend/sirius-components-view-emf/src/test/java/org/eclipse/sirius/components/view/emf/view/DynamicDiagramsTests.java

```diff
@@ -130,7 +132,8 @@ private Diagram render(DiagramDescription diagramDescription, Object target) {
         View view = ViewFactory.eINSTANCE.createView();
         view.getDescriptions().add(diagramDescription);
 
-        ViewDiagramDescriptionConverter diagramDescriptionConverter = new ViewDiagramDescriptionConverter(new IObjectService.NoOp(), new IEditService.NoOp());
+        ViewDiagramDescriptionConverter diagramDescriptionConverter = new ViewDiagramDescriptionConverter(new IObjectService.NoOp(), new IEditService.NoOp(),
+                new StylesFactory(new ArrayList<>(), new IObjectService.NoOp()));
```

Comment:
StylesFactory has only one parameter

Reply:
`List.of()` instead of `new ArrayList()`. `new ArrayList()` is not immutable

Reply:
yes it is fixed